src/control.h

```diff
@@ -134,8 +135,8 @@ class Control {
 
   rak::priority_item  m_taskShutdown;
 
-  bool                m_shutdownReceived lt_cacheline_aligned;
-  bool                m_shutdownQuick lt_cacheline_aligned;
+  std::atomic<bool>   m_shutdownReceived lt_cacheline_aligned;
+  std::atomic<bool>   m_shutdownQuick lt_cacheline_aligned;
```

Comment:
These are cacheline aligned, so we should remove all instances of `lt_cacheline_aligned` when we're not doing `__sync`.